modules/javafx.swing/src/main/java/module-info.java

```diff
@@ -36,6 +36,7 @@
 module javafx.swing {
     requires java.datatransfer;
     requires javafx.base;
+    requires javafx.controls;
```

Comment:
This should be reverted. I can think of no reason for the `javafx.swing` module to depend on `javafx.controls`. Drag and drop is not a Controls action. It is a scene graph action.